ingestr/src/.dlt/.sources

```diff
@@ -0,0 +1,28 @@
+engine_version: 1
```

Comment:
we'd like to remove the whole `.dlt` folder from here as well

Reply:
ok